modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/AndesClient.java

```diff
@@ -1,589 +1,304 @@
-            log.info("host and port==>" + hostInformation);
-
-            //decode host and port
-            String hostName = hostInformation.split(":")[0];
-            String port = hostInformation.split(":")[1];
-
-            //browse and get the message count
-            int messageCount = browseQueue(hostName, port, this.username, this.password, destinations, Integer.MAX_VALUE, false);
-
-            //activate receiver for the queue(if specified more first one/ if specified more than one host get the first one)
-            //start a queue receiver
-            QueueMessageReceiver queueMessageReceiver = new QueueMessageReceiver
-                    ("", hostName, port, this.username, this.password, destinations, 1, true, queueMessageCounter, 0,
-                            Integer.MAX_VALUE, false, "", messageCount, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE);
-            queueListeners.add(queueMessageReceiver);
-            new Thread(queueMessageReceiver).start();
-
-            topicMessageCounter.set(messageCount);
-
-        } else if (mode.equals("analyse")) {
-
-            /**
-             * mode analyse
-             *     printMessages/printDuplicates/printMissing/printSorted/checkOrder/clearFile
-             */
-            String operation = analyticOperation;
-            String numberOfMessagesSentAsString = "";
-            int numberOfMessagesSent = 0;
-
-            if (operation.equals("printMissing")) {
-
-                numberOfMessagesSentAsString = numberOfMessagesExpectedForAnalysis;
-                numberOfMessagesSent = Integer.parseInt(numberOfMessagesSentAsString);
-            }
-
-            //print input information
-            log.info("mode==>" + mode);
-            log.info("operation==>" + operation);
-
-            AndesClientOutputParser andesClientOutputParser = new AndesClientOutputParser(filePathToWriteReceivedMessages);
-
-            if (operation.equals("printMessages")) {
-                andesClientOutputParser.printMessagesMap();
-            } else if (operation.equals("printDuplicates")) {
-                andesClientOutputParser.printDuplicateMessages();
-            } else if (operation.equals("printMissing")) {
-                andesClientOutputParser.printMissingMessages(numberOfMessagesSent);
-            } else if (operation.equals("printSorted")) {
-                andesClientOutputParser.printMessagesSorted();
-            } else if (operation.equals("checkOrder")) {
-                log.info("MESSAGE ORDER PRESERVED: " + andesClientOutputParser.checkIfMessagesAreInOrder());
-            } else if (operation.equals("clearFile")) {
-                andesClientOutputParser.clearFile();
-            } else {
-                log.info("analyse operation not found...");
             }
         } else {
-            log.info("ERROR: Unknown mode of operation");
+            throw new AndesClientException("The amount of subscribers cannot be less than 1");
```

Comment:
Good to add the input subscriber count so someone can figure out whats wrong from the log.